main.ts

```diff
@@ -969,7 +1101,186 @@ const issueCommand = new Command()
+  )
+  .option(
+    "--no-use-default-template",
+    "Do not use default template for the issue",
+  )
+  .option("--no-color", "Disable colored output")
+  .arguments("[title:string]")
+  .action(
+    async (
+      {
+        start,
+        assignee,
+        dueDate,
+        useDefaultTemplate,
+        parent,
+        priority,
+        estimate,
+        labels,
+        team,
+        project,
+        dryRun,
+        color,
+      },
+      title,
+    ) => {
+      const showSpinner = color && Deno.stdout.isTerminal();
+      const spinner = showSpinner ? new Spinner() : null;
+      spinner?.start();
+      try {
+        const teamId = (team === undefined)
+          ? await getTeamId()
+          : team.toUpperCase();
+        const teamUid = await getTeamUid(teamId, spinner);
+        if (start && assignee === undefined) {
+          assignee = "self";
+        }
+        if (start && assignee !== undefined && assignee !== "self") {
+          console.error("Cannot use --start and a non-self --assignee");
+        }
+        let assigneeId = undefined;
+        if (assignee !== undefined) {
+          try {
+            // try with display name first
+            assigneeId = await getUserId(assignee, spinner);
+          } catch (_) {
+            assigneeId = await tryPartialMatch(
+              "user",
+              "name",
+              assignee,
+              true,
+              spinner,
+            );
+          }
+        }
+        const labelIds: string[] = [];
+        if (labels !== undefined && labels !== true && labels.length > 0) {
+          // sequential in case of questions
+          for (const label of labels) {
+            labelIds.push(await getIssueLabelId(label, spinner));
+          }
+        }
+        const projectId = (project !== undefined)
+          ? await getProjectId(project, spinner)
+          : undefined;
+        const parentId = (parent !== undefined)
+          ? await getIssueId(parent)
+          : undefined;
+        if (dueDate !== undefined) {
+          const parsed = new Date(dueDate);
+          if (isNaN(parsed.getTime())) {
+            console.error("Invalid due date format");
+            Deno.exit(1);
+          }
+          dueDate = parsed.toISOString().substring(0, 10);
+        }
+
+        const query = `
+          mutation createIssue($title: String!, $assigneeId: String, $dueDate: TimelessDate, $parentId: String, $priority: Int, $estimate: Int, $labelIds: [String!], $teamId: String!, $projectId: String, $useDefaultTemplate: Boolean) {
+            issueCreate(input: {
+              title: $title
+              assigneeId: $assigneeId
+              dueDate: $dueDate
+              parentId: $parentId
+              priority: $priority
+              estimate: $estimate
+              labelIds: $labelIds
+              teamId: $teamId
+              projectId: $projectId
+              useDefaultTemplate: $useDefaultTemplate
+            }) {
```

Comment:
can this be more simply:

```
mutation createIssue($input: IssueCreateInput!) {
  issueCreate(input: $input) { ... }
}

const data = await fetchGraphQL(query, { input: { title, assigneeId, ... } })
```

?

the variables might be a different shape when you call it, but this will be a bit cleaner i think